logging.js

```diff
@@ -235,11 +238,28 @@ export class LoggingClient {
 	}
 
 	_throttle(log) {
+
+		const now = Date.now();
+
+		// rate limit number of errors to 100 per minute
+		let sliceIndex = this._logTimestamps.length;
```

Comment:
Did you mean to switch to this? I personally like it lol, but based on the convo with @oelhanafey sounds like you didn't 😛

Reply:
Haha oh no! I built it locally and then accidentally committed it when I renamed `_logTimestamps`. Thanks for catching it!